zhaquirks/tuya/ts110e.py

```diff
@@ -0,0 +1,142 @@
+        if attrid == TUYA_LEVEL_ATTRIBUTE:
+            value = (value * 255) // 1000
+            attrid = 0x0000
+
+        super()._update_attribute(attrid, value)
+
+    async def command(
+        self,
+        command_id: Union[foundation.Command, int, t.uint8_t],
+        *args,
+        manufacturer: Optional[Union[int, t.uint16_t]] = None,
+        expect_reply: bool = True,
+        tsn: Optional[Union[int, t.uint8_t]] = None,
+    ):
+        """Override the default Cluster command."""
+        self.debug(
+            "Sending Cluster Command. Cluster Command is %x, Arguments are %s",
+            command_id,
+            args,
+        )
+        # move_to_level, move, move_to_level_with_on_off
+        if command_id in (0x0000, 0x0001, 0x0004):
+            # convert dim values to 0-1000
+            brightness = (args[0] * 1000) // 255
+            return await super().command(
+                TUYA_CUSTOM_LEVEL_COMMAND,
+                TuyaLevelPayload(level=brightness, transtime=0),
+                manufacturer=manufacturer,
+                expect_reply=expect_reply,
+                tsn=tsn,
+            )
+
+        return super().command(command_id, *args, manufacturer, expect_reply, tsn)
+
+
+class DimmerSwitchWithNeutral1Gang(TuyaDimmerSwitch):
+    """Tuya Dimmer Switch Module With Neutral 1 Gang."""
+
+    signature = {
+        MODELS_INFO: [("_TZ3210_ngqk6jia", "TS110E")],
+        ENDPOINTS: {
+            # <SimpleDescriptor endpoint=1 profile=260 device_type=257
+            # input_clusters=[0, 4, 5, 6, 8, 57345]
+            # output_clusters=[10, 25]>
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.DIMMABLE_LIGHT,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    LevelControl.cluster_id,
+                    TuyaZBExternalSwitchTypeCluster.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [Time.cluster_id, Ota.cluster_id],
+            },
+            242: {
+                # <SimpleDescriptor endpoint=242 profile=41440 device_type=97
+                # input_clusters=[]
+                # output_clusters=[33]
+                PROFILE_ID: 41440,
+                DEVICE_TYPE: 97,
+                INPUT_CLUSTERS: [],
+                OUTPUT_CLUSTERS: [GreenPowerProxy.cluster_id],
+            },
+        },
+    }
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.DIMMABLE_LIGHT,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    F000LevelControlCluster,
+                    TuyaZBExternalSwitchTypeCluster.cluster_id,
```

Comment:
remove the '.cluster_id' part:
```python
                    TuyaZBExternalSwitchTypeCluster,
```

Reply:
I think first testing if the cluster / attribute is working on the device or not and is its not working then deleting it in the replacement (its always possible adding it later if its needed) but its very likely not working on this device.

The implementation is here:
https://github.com/zigpy/zha-device-handlers/blob/8ae597d335e2f19f73fdc82f78fb5ae5fb18a40c/zhaquirks/tuya/__init__.py#L887-L902

Reply:
I actually noticed that I had to leave in the `.cluster_id` part, otherwise the quirk wouldn't show up on the device anymore.